.github/workflows/release-image-builds.yml

```
@@ -96,13 +88,13 @@ jobs:
       - name: Check out docker-xcache
         uses: actions/checkout@v2
       - name: Create dummy Dockerfile
-        run: echo "FROM docker.pkg.github.com/$GITHUB_REPOSITORY/${{ matrix.image }}:${{ matrix.yum_repo_prefix }}minefield-${{ needs.get-timestamp.outputs.dtag }}" > Dockerfile
+        run: echo "FROM docker.pkg.github.com/$GITHUB_REPOSITORY/${{ matrix.image }}:upcoming-minefield-${{ needs.get-timestamp.outputs.dtag }}" > Dockerfile
       - run: echo ${{ secrets.GITHUB_TOKEN }} | docker login docker.pkg.github.com -u $GITHUB_ACTOR --password-stdin
-      - run: docker pull docker.pkg.github.com/$GITHUB_REPOSITORY/${{ matrix.image }}:${{ matrix.yum_repo_prefix }}minefield-${{ needs.get-timestamp.outputs.dtag }}
+      - run: docker pull docker.pkg.github.com/$GITHUB_REPOSITORY/${{ matrix.image }}:upcoming-minefield-${{ needs.get-timestamp.outputs.dtag }}
       - name: Build ${{ matrix.image}} image
         uses: docker/build-push-action@v1
         with:
           repository: opensciencegrid/${{ matrix.image }}
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
-          tags: ${{ matrix.yum_repo_prefix }}fresh, ${{ matrix.yum_repo_prefix }}${{ needs.get-timestamp.outputs.dtag }}
+          tags: fresh, upcoming-${{ needs.get-timestamp.outputs.dtag }}
```

```suggestion
          tags: fresh, ${{ needs.get-timestamp.outputs.dtag }}
```

I think we should only ship `fresh` and timestamp tags

---

Oops, didn't see that.